libs/ui/src/lib/ui-dialog/ui-dialog.component.ts

```diff
@@ -0,0 +1,9 @@
+import { Component, ViewEncapsulation } from '@angular/core';
+
+@Component({
+  selector: 'ui-dialog',
+  templateUrl: './ui-dialog.component.html',
+  styleUrls: ['./ui-dialog.component.scss'],
+  encapsulation: ViewEncapsulation.None
```

Comment:
Is this necessary?

Reply:
This is indeed necessary to style the inner content of the `mat-card > mat-card-content` MD component.

Reply:
If you need to use it make sure you wrap all of your scss rules within with
`ui-dialog { ... }` so that the rules don't leak into other component's stylings